assets/js/app.js

```diff
@@ -16,26 +16,37 @@
 //
 
 // Include phoenix_html to handle method=PUT/DELETE in forms and buttons.
-import "phoenix_html"
+import "phoenix_html";
 // Establish Phoenix Socket and LiveView configuration.
-import {Socket} from "phoenix"
-import {LiveSocket} from "phoenix_live_view"
-import topbar from "../vendor/topbar"
+import { Socket } from "phoenix";
+import { LiveSocket } from "phoenix_live_view";
+import topbar from "../vendor/topbar";
+import ImageCropper from "./hooks/cropper";
 
-let csrfToken = document.querySelector("meta[name='csrf-token']").getAttribute("content")
-let liveSocket = new LiveSocket("/live", Socket, {params: {_csrf_token: csrfToken}})
+let Hooks = {};
+
+Hooks = {
+  ImageCropper,
```

Comment:
When I upload a non square photo I don't see an ImageCropper anywhere. Is that a bug?

Reply:
Let's remove the ImageCropper functionality.

Reply:
> When I upload a non square photo I don't see an ImageCropper anywhere. Is that a bug?

This is fixed. You can try with a non-square image